Mastel22/weather-app
# #169259198 User can view the weather for the next five days

#### What does this PR do?
This pull request adds a 5-day weather forecast component to the application

#### Description of Task to be completed?
- [x] Have the 5-day weather component rendering correctly with the weather forecast details
- [x] Style the component

#### How should this be manually tested?
- After cloning the repo 'cd' into it
- run 'npm install' to install all the dependencies
- run 'npm start' to start the server
You should see the weather-app loaded on the browser with the 5-day weather forecast rendered with the weather data

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#169259198

#### Screenshots (if appropriate)
<img width="1440" alt="5dayweather" src="https://user-images.githubusercontent.com/17108099/67330063-131bf700-f51c-11e9-888d-30b7b1ee4240.png">

#### Questions:
N/A
